platform/pulpcore/app/status.py

```diff
@@ -0,0 +1,65 @@
+"""
+Module to return status information about a running Pulp instance
```

Comment:
Why the helper functions in a separate module?  Why not just implement in the StatusView?  I like the modularity but seems these functions should be protected (and perhaps static) methods on the StatusView class itself.